integreat_cms/cms/models/users/user.py

```diff
@@ -219,3 +219,9 @@ class Meta:
         default_permissions = ("change", "delete", "view")
         #: The fields which are used to sort the returned objects of a QuerySet
         ordering = ["username"]
+        #: Permissions that are not related to real models
+        permissions = (
+            ("view_translation_report", "view_translation_report"),
+            ("view_broken_links", "view_broken_links"),
+            ("view_statistics", "view_statistics"),
```

Comment:
I mean that's open for debate, but if we're using permission objects to hide menu items, we could also treat them like "real" permissions and also deny access to the analytics views itself... otherwise, users can still view these pages by typing in the link directly... (in theory, this shouldn't be a problem, I'm just wondering if we should take these measures to have a consistent behaviour when it comes to permissions.)

Reply:
I think it's anyway nice to have, so implemented 🙂